src/EventStore.ClusterNode/Program.cs

```diff
@@ -30,7 +30,8 @@ public class Program : ProgramBase<ClusterNodeOptions>
 
         public static void Main(string[] args)
```

Comment:
There is one instance of where I applied the null-conditional operator (line 122). Do you still want me to revert all the formatting changes?

To clarify, I didn't go through the whole file changing all spaces and same-line brackets... I just hit Ctrl K + D which applies the formatting.